src/metrics.jl

```diff
@@ -289,6 +293,10 @@ end
 end
 rogerstanimoto{T <: Bool}(a::AbstractArray{T}, b::AbstractArray{T}) = evaluate(RogersTanimoto(), a, b)
 
+
+# Utility
+sqr(x) = x*x
```

Comment:
Yeah, I'm not sure under what circumstances it falls back to `power_by_squaring`. I'd say go with the simpler `^2` for now and we can revisit it if we find that things are slooooow, if that seems reasonable.